providers/hetznercloud/provider.go

```diff
@@ -161,14 +160,48 @@ func (d *Provider) DeployAgent(ctx context.Context, agent *woodpecker.Agent) err
 			EnableIPv4: d.enableIPv4,
 			EnableIPv6: d.enableIPv6,
 		},
-	})
+	}
+
+	_, _, err = d.client.Server().Create(ctx, serverCreateOpts)
```

Comment:
You could append the normal server type to the array as first elemwnt here fo simplify the code a bit.

Reply:
Good point. Still not sure about the UX issue discussed above. Do you have an opinion about it?